src/JSInterop/Microsoft.JSInterop.JS/src/yarn.lock

```diff
@@ -0,0 +1,258 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
@JunTaoLuo does the package.json need to be removed?

Reply:
You mean the package.lock.json? Then you can remove it if you'd like.

Reply:
Yeah, you're right, the lock file.